### PR TITLE
Update namespace prefixes of AST classes

### PR DESCRIPTION
**Pull request by Stagno**
## Technical Description

To update to dawn's new namespace organization. AST nodes and utils needed are now in namespace dawn::sir.

#### Example

`dawn::IfStmt` becomes `dawn::sir::IfStmt`

## Dependencies

MeteoSwiss-APN/dawn#246

### CODE REVIEW

**Comment by Stagno** [comment]
launch pipeline

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -90,8 +90,8 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
-    GIT_TAG "master" 
+    GIT_REPOSITORY "git@github.com:Stagno/dawn.git"
+    GIT_TAG "refactor_AST"
```
Comment (twicki) [inline bundle/CMakeLists.txt]: remember to change this back

Comment (Stagno) [reply bundle/CMakeLists.txt]: launch pipeline

**Comment by Stagno** [comment]
launch pipeline

**Comment by Stagno** [comment]
launch pipeline

**Pull request merged**